app/src/main/java/io/novafoundation/nova/app/root/navigation/navigators/Navigator.kt

```diff
@@ -482,7 +482,7 @@ class Navigator(
 
         navigationBuilder().action(R.id.action_openUnifiedAddressDialog)
             .setArgs(ChainAddressSelectorFragment.getBundle(payload))
-            .navigateInFirstAttachedContext()
+            .navigateInRoot()
```

Comment:
Still, we are missing unified "when to use what" guide that will help us to avoid navigation issues entirely. Lets leave a comment for  for `navigationBuilder` and `navigateInFirstAttachedContext` / `navigateInRoot` that will also describe how actions should be added in xcml e.t.c

Like imagine a new person reading this code. They will for sure ask a question "why do we use navigateInRoot here and navigateInFirstAttachedContext in other places?". Given that we didnt manage to create a clear system from the beginning, lets address it with proper docs at the very least